node/cn/handler_test.go

```diff
@@ -47,9 +47,13 @@ var addrs []common.Address
 var keys []*ecdsa.PrivateKey
 var nodeids []discover.NodeID
 
-var tx *types.Transaction
+var tx1 *types.Transaction
```

Comment:
tx1? Will you have tx2 and tx3 later?

Reply:
So don't people usually use arrays in this case? Why do we create a single variable and add a number to the name?

Reply:
You're right. I will use a slice if needed. In this case, while creating test, I used `tx2` but removed it since it was not needed anymore. I'lll reflect your comment in next PR.